src/forms/components/page/ohri-form-page.tsx

```diff
@@ -19,21 +20,23 @@ function OHRIFormPage({ page, onFieldChange, setSelectedPage, isCollapsed }) {
         </div>
         <Accordion>
           {/* <p className={styles.required}>All fields are required unless marked optional</p> */}
-          {page.sections.map((sec, index) => {
-            return (
-              <AccordionItem title={sec.label} open={isCollapsed} className={styles.sectionContent}>
-                <div className={styles.formSection} key={index}>
-                  <OHRIFormSection
-                    fields={sec.questions}
-                    showTitle={page.sections.length > 1}
-                    onFieldChange={onFieldChange}
-                    sectionTitle={sec.label}
-                    key={index}
-                  />
-                </div>
-              </AccordionItem>
-            );
-          })}
+          {page.sections
+            .filter(sec => !isTrue(sec.isHidden))
```

Comment:
Would this not work without `isTrue`?

Reply:
Well, there cases were we have values like:

```json

{
   isHidden: "true"
}
```

Basically boolean values set as strings. So that's a boolean util to cater for that. I got to learn that this comes from Ampath's form designer. Here is an example: https://github.com/UCSF-IGHS/openmrs-esm-ohri/blob/master/src/packages/hiv/forms/hts/2.0.json#L9

Reply:
@samuelmale @eudson merging this in. We can follow the discussion on the same thread if need be.